### ISSUE TITLE
Updates needed for Slack channel

### ISSUE THREAD

**Issue by annethyme** [act-rules/act-rules.github.io#617]
The Slack channel for this Community Group is still called Auto-WCAG. It should be renamed, if possible.

And then maybe we should write an "onboarding checklist" on the website that tells how to join all channels when joining the group, so we don't forget to invite new members to our slack channel?

**Comment by kasperisager** [comment]
We might also want to consider open something slightly less walled, such as https://spectrum.chat/?

**Comment by annethyme** [comment]
@WilcoFiers renames the Slack channel.

Decided on call today to add info on Slack channel to Contribute page (https://act-rules.github.io/pages/contribute/) under "Join the Community Group".

**Comment by jkodu** [comment]
This work has been completed.
